src/Support/Date.php

```diff
@@ -34,42 +34,31 @@ public static function howLongAgo(string $date) : string
         $hour = $minute * 60;
         $day = $hour * 24;
         $month = $day * 30;
+        $years = $day * 360;
```

Comment:
```suggestion
        $years = $day * 365;
```

Should it not be 365?